install/bootstrap_livecd_usb.md

```diff
@@ -723,19 +722,20 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
 
    ```bash
    pit# cd /var/www/ephemeral
-   pit# export CSM_RELEASE=csm-x.y.z
-   pit# echo $CSM_RELEASE
-   pit# export CSM_PATH=$(pwd)/${CSM_RELEASE}
-   pit# echo $CSM_PATH
+   pit:/var/www/ephemeral# export CSM_RELEASE=csm-x.y.z
+   pit:/var/www/ephemeral# echo $CSM_RELEASE
+   pit:/var/www/ephemeral# export CSM_PATH=$(pwd)/${CSM_RELEASE}
+   pit:/var/www/ephemeral# echo $CSM_PATH
    ```
 
 1. Install Goss Tests and Server
 
    The following assumes the CSM_PATH environment variable is set to the absolute path of the unpacked CSM release.
 
    ```bash
-   pit# rpm -Uvh --force $(find ${CSM_PATH}/rpm/embedded/cray/csm/ -name "goss-servers*.rpm" | sort -V | tail -1)
-   pit# rpm -Uvh --force $(find ${CSM_PATH}/rpm/cray/csm/ -name "csm-testing*.rpm" | sort -V | tail -1)
+   pit:/var/www/ephemeral# rpm -Uvh --force $(find ./${CSM_RELEASE}/rpm/ -name "goss-servers*.rpm" | sort -V | tail -1)
+   pit:/var/www/ephemeral# rpm -Uvh --force $(find ./${CSM_RELEASE}/rpm/ -name "csm-testing*.rpm" | sort -V | tail -1)
+   pit:/var/www/ephemeral# cd 
    ```
```

Comment:
This duplicates the logic here: https://github.com/Cray-HPE/csm/blob/main/lib/install-goss-tests.sh